.gitignore

```diff
@@ -39,3 +39,10 @@ changes/1024/
 # IntelliJ
 .idea/
 *.iml
+models-yamtl/
+solutions/EMFSolutionYAMTL_element/
+solutions/EMFSolutionYAMTL_helpersOnly_v1/
+solutions/EMFSolutionYAMTL_indexed
+solutions/EMFSolutionYAMTL_uf
+**/*.documented.xmi
+models/1024/
```

Comment:
It is intentional to have the .gitignore in models/1024 folder to make sure that the folder exists.
Can I put it back?